lib/tasks/search.rake

```diff
@@ -104,6 +104,13 @@ namespace :search do
     end
   end
 
+  desc "Update search index for all political content"
+  task political: :environment do
+    Edition
+      .where(political: true, state: "published")
+      .find_each { |edition| edition&.update_in_search_index }
```

Comment:
I found it confusing that the `&.` operator applies to the right hand side as well as the left hand side! But since it's just `edition.try(:update_in_search_index)` I suppose that makes sense.